docs/integrations/saas-cloud/cisco-meraki-c2c.md

```diff
@@ -8,7 +8,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('/img/integrations/security-threat-detection/ciscomeraki.png')} alt="ciscomeraki-logo" width="75"/>
 
-The Sumo Logic app for Cisco Meraki provides real-time insights into the events and helps you to identify potential network events along with admin activities. This app can effectively manage and optimize your network performance, enhance security, and proactively respond to potential threats. The comprehensive insights and monitoring capabilities enable efficient network administration and contribute to a robust and secure network infrastructure.
+The Sumo Logic app for Cisco Meraki C2C provides real-time insights into the events and helps you to identify potential network events along with admin activities. This app can effectively manage and optimize your network performance, enhance security, and proactively respond to potential threats. The comprehensive insights and monitoring capabilities enable efficient network administration and contribute to a robust and secure network infrastructure.
```

Comment:
```suggestion
The Sumo Logic app for Cisco Meraki provides real-time insights into the events and helps you to identify potential network events along with admin activities. This app can effectively manage and optimize your network performance, enhance security, and proactively respond to potential threats. The comprehensive insights and monitoring capabilities enable efficient network administration and contribute to a robust and secure network infrastructure.
```

Reply:
We should link Cisco Meraki C2C here.

Reply:
@JainM6 The app name itself is Cisco Meraki C2C. I had a discussion with dev guys and they wanted the app name in this way just to differentiate this app from the existing Cisco Meraki (uses Syslog source) and this app uses Sumo Logic C2C source.

Reply:
@JV0812 I had a follow-up question. Should it be `Cisco Meraki - C2C` or `Cisco Meraki C2C`. I see both formats in this doc.